keps/sig-auth/1205-bound-service-account-tokens/README.md

```diff
@@ -265,53 +258,161 @@ service account token on behalf of pods running on that node. The
-audit report that stale or legacy token are in use. The operator can turn off 
-the command line flag after no stale token is being used.
+### Graduation Criteria
 
+#### Alpha->Beta
 
+All known migration frictions have been fixed.
 
-### Graduation Criteria
+- PodSecurityPolicies that allow secrets but not projected volumes will
+  handicap the provision of token volumes.
+  - Fixed in https://github.com/kubernetes/kubernetes/pull/92006
+- In-cluster clients that don’t reload service account tokens will start
+  failing an hour after deployment.
+  - Fixed in https://github.com/kubernetes/kubernetes/issues/68164
+- Pods running as non root may not access the service account token.
+  - Fixed in https://github.com/kubernetes/kubernetes/pull/89193
 
 #### Beta -> GA Graduation
 
-- TBD
+New `ServiceAccount` admission controller WAI in Beta for >= 1 minor without
+significant issues.
+
+## Production Readiness Review Questionnaire
+
+### Feature Enablement and Rollback
+
+- **How can this feature be enabled / disabled in a live cluster?**
+
+  - Feature gate name: `BoundServiceAccountVolume`
+  - Components depending on the feature gate: kubelet, kube-apiserver
+  - Will enabling / disabling the feature require downtime of the control
+    plane? yes, need to restart kube apiserver.
+  - Will enabling / disabling the feature require downtime or reprovisioning
+    of a node? yes, need to restart kubelet.
+
+- **Does enabling the feature change any default behavior?** yes, pods'
+  service account tokens will not be long-lived and are not stored as Secrets
+  any more.
+
+- **Can the feature be disabled once it has been enabled (i.e. can we roll
+  back the enablement)?** yes.
+
+- **What happens if we reenable the feature if it was previously rolled
+  back?** the same as the first enablement.
+
+- **Are there any tests for feature enablement/disablement?** yes, unit tests
+  will cover this.
+
+### Scalability
+
+- **Will enabling / using this feature result in any new API calls?**
+
+  - API call type: `TokenRequest`
+  - estimated throughput: 1/pod.
+  - originating component: kubelet
+  - components listing and/or watching resources they didn't before: N/A.
+  - API calls that may be triggered by changes of some Kubernetes resources:
+    N/A.
+  - periodic API calls to reconcile state (e.g. periodic fetching state,
+    heartbeats, leader election, etc.): N/A.
+
+- **Will enabling / using this feature result in introducing new API types?**
+  no.
+
+- **Will enabling / using this feature result in any new calls to the cloud
+  provider?** no.
+
+- **Will enabling / using this feature result in increasing size or count of
+  the existing API objects?** no.
+
+- **Will enabling / using this feature result in increasing time taken by any
+  operations covered by [existing SLIs/SLOs]?** no.
+
+- **Will enabling / using this feature result in non-negligible increase of
+  resource usage (CPU, RAM, disk, IO, ...) in any components?** no.
```

Comment:
it adds a token minting operation in the API server every ~8 minutes for every pod

Reply:
done